addons/im_livechat/static/tests/sidebar_patch.test.js

```diff
@@ -352,7 +352,7 @@ test("Clicking on unpin button unpins the channel", async () => {
     await start();
     await openDiscuss();
     await click(".o-mail-DiscussSidebarChannel [title='Unpin Conversation']");
-    await contains(".o_notification", { text: "You unpinned your conversation with Visitor 11" });
+    await contains(".o-mail-DiscussSidebarChannel", { count: 0, text: "Visitor 11" });
```

Comment:
Check selector positive assertion before unpin, so the selector makes sense (this doesn't pass because this is outdated selector)